storage/src/mmr/mem.rs

```diff
@@ -0,0 +1,854 @@
+//! A bare-bones MMR structure without pruning and where all nodes are hashes & maintained in
+//! memory.
+//!
+//! TERMINOLOGY:
```

Comment:
nit: use `# Terminology` for consistency with other documentation

Reply:
done